catalog/imaging/imaging-plugin-nitf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -14,17 +14,29 @@
  -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="Nitf Imaging Input Transformer"
-         id="NITF_Input_Transformer">
+    <OCD name="Nitf Render Plugin"
+         id="NITF_Render_Plugin">
         <AD
                 description="Maximum length of longest side of NITF overview image in pixels. The input transformer will calculate the size of the shorter side so that the overview will have the same aspect ratio as the original."
                 name="Overview image maximum side length (pixels)" id="maxSideLength" required="true"
                 type="Integer" default="1024"/>
+        <AD
+                description="Maximum NITF file size in Megabytes to create thumbnails/overviews for. Rendering large NITFs is memory intensive. Make sure your system is configured with enough memory to handle `<NITF-Size>*<Number-Of-Ingest-Threads>*4` Megabytes."
```

Comment:
`to create thumbnails/overviews for` -> `of which to create thumbnails/overviews`